bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/handler/AbstractNetatmoThingHandler.java

```diff
@@ -137,16 +132,39 @@ protected State getNAThingProperty(String channelId) {
     }
 
     protected void updateChannels() {
-        getThing().getChannels().stream().filter(channel -> channel.getKind() != ChannelKind.TRIGGER)
+        updateNonTriggerChannels();
+
+        triggerTriggerChannels();
```

Comment:
Maybe triggerEventChannels() and updateDataChannels() would be clearer ?

Reply:
ok, done